.vuepress/configs/sidebar/en.ts

```diff
@@ -137,6 +137,7 @@ export const sidebarEn: SidebarConfig = {
         'help',
         'system',
         'parsing',
+        'foreign_shell_scripts',
```

Comment:
Note: I placed this here because it seemed like a good follower to the `system` and `parsing` documents, but I'm totally fine with it being moved to the bottom or somewhere else.